cic/mixins.spec

```diff
@@ -26,3 +26,4 @@ camera-ext: ext-camera-only
 bluetooth: cic
 trusty: true(ref_target=cic)
 storage: sdcard-mmc0-usb-sd(adoptablesd=false,adoptableusb=false)
+avb: true
```

Comment:
Do we try to keep flags inside Kernelflinger? Is it possible in this case?

Reply:
No.  The flag is defined by Android, not kernelflinger. It is used to control to generate vbmeta image and sign images by AOSP code.  So should not put into kernelflinger.